storage/innobase/fil/fil0crypt.cc

```diff
@@ -1474,6 +1534,40 @@ static bool fil_crypt_space_needs_rotation(rotate_thread_t *state,
   return false;
 }
 
+bool fil_crypt_exclude_from_rotation(fil_space_t *space) {
```

Comment:
This is a mainly duplicated function, please refactor.

Reply:
It was a leftover. Removed it.Thanks for spotting this.